tests/test_job.py

```diff
@@ -2,6 +2,8 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
+import dill
```

Comment:
For tests, just use `json` so we don't need to pull in an external library just for this.